### PR TITLE
Drop the incubator- prefix for GitHub Repos

### PR DESCRIPTION
**Pull request by tisonkun**
According to the vote and related discussion - https://lists.apache.org/thread/1kf8spl8z8m91773yqzxbhx27mrhxhdg

### CODE REVIEW

File: pages/guides/lists.ad [hunk 1]
```diff
@@ -30,7 +30,6 @@ Please read the link:http://www.apache.org/foundation/public-archives.html[Apach
 This email list is where general and public discussions related to the Incubator take place.
 - link:mailto:general-subscribe@incubator.apache.org[Subscribe]
 - link:mailto:general-unsubscribe@incubator.apache.org[Unsubscribe]
-- link:http://mail-archives.apache.org/mod_mbox/incubator-general/[Archives]
```
Comment (tisonkun) [inline pages/guides/lists.ad:33]: This domain is now an alias to `https://lists.apache.org/list.html/...`. Ditto others.

**Review by justinmclean — APPROVED** [review]
Looks good to me

**Comment by tisonkun** [comment]
Thanks for your review @justinmclean! Merging ...

Instead I'd try to add a sentence about displaying the incubating status in README and repo description. Perhaps in the DISCLAIMER session.

**Comment by Humbedooh** [comment]
I will note that this change has not gone into effect yet, as the incubator hasn't formally requested any changes to the repository procurement process at present. It probably should :)

**Comment by tisonkun** [comment]
> the repository procurement process at present

May I ask what is the process from INFAR team point of view? I may know some practices and have the approximate knowledge, but yet to find a document or where I can send the request to move forward.

**Comment by Humbedooh** [comment]
I'd suggest requesting the change via the infra mailing list.

**Pull request merged**